schema-extension.md

```diff
@@ -0,0 +1,162 @@
+# CAR schema extension
+
+The CAR schema can be extended by creating `get_schema_extension` method in `app.py` file of the connector and returning `SchemaExtension` object.
+Example can found in [reference connector](./connectors/reference_connector/app.py)
+
+`SchemaExtension` object constructor needs following variables:
+- \<UUID\> is the ID of the extension. It is unique for every extension and it never changes.
+- \<Connector Name\> is a name that makes it easier to see where this extension comes from.
+- \<Version Number\> is an integer number encoded as a string; whenever an extension is changed this number needs to be incremented.
```

Comment:
Is the version number of the extension or the target CAR schema?

Reply:
Its version of extension, I'll update the description to make it clearer.